modules/space/rendering/renderablerings.cpp

```diff
@@ -234,29 +229,25 @@ void RenderableRings::update(const UpdateData& data) {
 }
 
 void RenderableRings::loadTexture() {
-    if (!_texturePath.value().empty()) {
-        using namespace ghoul::io;
-        using namespace ghoul::opengl;
-        std::unique_ptr<Texture> texture = TextureReader::ref().loadTexture(
-            absPath(_texturePath),
-            1
+    using namespace ghoul::io;
+    using namespace ghoul::opengl;
+    std::unique_ptr<Texture> texture = TextureReader::ref().loadTexture(
+        _texturePath.value(),
+        1
+    );
+
+    if (texture) {
+        LDEBUGC(
+            "RenderableRings",
+            std::format("Loaded texture from '{}'", absPath(_texturePath))
```

Comment:
Why keep the abspath here but not above? I.e. in `loadTexture`

Reply:
Oversight.  `_texturePath` is a StringProperty and we don't expect people to enter path tokens into the UI, so I removed the bottom `absPath` as well.

Once the UI rewrite is done, I'd like to start working on a `FilePath` view for the StringProperty to properly represent that this property expects a file and not just any string